src/main/java/emu/grasscutter/server/packet/recv/HandlerGetPlayerTokenReq.java

```diff
@@ -119,7 +120,7 @@ public void handle(GameSession session, byte[] header, byte[] payload) throws Ex
                 var client_seed = ByteBuffer.wrap(cipher.doFinal(client_seed_encrypted)).getLong();
 
```

Comment:
you can probably make a new variable for storing the encryption seed rather than use `? :` twice